jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -591,6 +591,12 @@ public void beforeTaskUpdatedEvent(TaskEvent event) {
         
     }
 
+    public String getUpdateFieldLog(String fieldName, String previousValue, String value){
+        return "Updated "+ fieldName + " {"+
+                (previousValue != null? "From: '"+ previousValue + "' to: " :  "" ) +
+                "'"+ (value != null ? value :  "" ) + "'}" ;
```

Comment:
Correct me if I am wrong, but in case previousValue is null and new value is 'value' it will output:
`Updated fieldName {'value'}`. Don't we want to include the 'From... to' clause in all cases?